CONTRIBUTING.md

```diff
@@ -31,7 +31,7 @@ SECRET_KEY=YOUR_SECRET_KEY
 
 ### Target frameworks
 
-By default the tests will run agaisnt runtimes .Net Core 3.1, .Net 5.0, and .Net 6.0
```

Comment:
Since your changing this, the versions listed is out of date, its should be `.Net 6.0 and .Net 7.0`

Reply:
Did you want this changed in all places of the file?

Reply:
Yes, every place it is applicable